nixos/tests/all-tests.nix

```diff
@@ -344,6 +344,7 @@ in {
   hedgedoc = handleTest ./hedgedoc.nix {};
   herbstluftwm = handleTest ./herbstluftwm.nix {};
   homepage-dashboard = handleTest ./homepage-dashboard.nix {};
+  honk = handleTest ./honk.nix {};
```

Comment:
The manual recommends using `runTest` here and dropping the import of the `python-testing` module in the test file. You just define the test module in the test file, without wrapping it in any function call, and then call `runTest` on it.

Reference to the manual: https://nixos.org/manual/nixos/stable/#sec-calling-nixos-tests

Reply:
Done! Thanks for noticing me, I wasn't aware of this.